apps/examples/testcase/le_tc/kernel/tc_libc_math.c

```diff
@@ -4213,21 +4213,33 @@ static void tc_libc_math_ynf(void)
  */
 static void tc_libc_math_gamma(void)
 {
-	double value;
-	double result;
-
-	value = 2.0;
-	result = gamma(value);
-	TC_ASSERT_EQ("gamma", result, 1);
-
-	value = 3.0;
-	result = gamma(value);
-	TC_ASSERT_EQ("gamma", result, 2);
-
-	value = INFINITY;
-	result = gamma(value);
-	TC_ASSERT_EQ("gamma", result, (double)INFINITY);
-
+	double in_val[13] = { 2.0, 3.0, INFINITY, 256.5,
+						1.0e-20, -257.5, -256.5, -1.0,
+						-1.5, 8.1, 8.3, 8.8, 9.3 };
+	double sol_val[13] = { 1, 2, INFINITY, 0x1p1023 * 256.5,
+						1 / 1.0e-20, 0, -0.0, NAN,
```

Comment:
Please use ZERO macro instead of 0 and -ZERO instead of -0.0

Reply:
modify